src/gui/mainwindow.h

```diff
@@ -196,6 +197,8 @@ class MainWindow : public wxFrame, public WindowAttributesPickle
         static const unsigned int OPT_PAGE_GROUPS   = 4;
 
 	const wxArrayString& GetTabNames();
+	
+	static const unsigned int mySHOW_ERROR_MESSAGE = 10000;
```

Comment:
But there is no guarantee some one useses same $unique_number elsewhere in code in future